spec/index.bs

```diff
@@ -3300,9 +3301,19 @@ GPUCommandBuffer includes GPUObjectBase;
 
 <script type=idl>
 dictionary GPUCommandBufferDescriptor : GPUObjectDescriptorBase {
+    boolean measureExecutionTime = false;
```

Comment:
I realized in the meeting that we need to move this to GPUCommandEncoderDescriptor. This is most likely required for some implementations so they can know to enqueue the "timer start" command at the beginning of encoding.

Reply:
My thought is we can insert timestmap query in `CommandBuffer::RecordCommands` if we `measureExecutionTime` is enabled in `GPUCommandBufferDescriptor`. 

> This is most likely required for some implementations

I'm not sure if you mean Metal backend which has three types of encoders. For Metal, we can read the GPUStartTime and GPUEndTime directly from command buffer when it's completed, so it doesn't need this flag.

Reply:
Think of Vulkan in a WebGPU implementation that tries to record commands live as they come. We'll need to insert a timestamp at the beginning of the `VkCommandBuffer`. For this, we need the flag on the `GPUCommandEncoderDescriptor`

Reply:
Right, we want to leave implementations open to record the backing command buffer on the fly, rather than all at once at the end like Dawn does.

Reply:
Thanks for the explanation.